packages/core/test/invoke.test.ts

```diff
@@ -118,16 +118,18 @@ describe('invoke', () => {
           start: {
             invoke: {
               src: 'child',
-              id: 'someService',
-              autoForward: true
+              id: 'someService'
```

Comment:
similarly, this test doesn't quite bring any value, I think - it's almost identical to the one below it

Reply:
Should we delete it?

Reply:
i think so